codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Connection.java

```diff
@@ -253,6 +253,30 @@
     }
 
     /**
+     * A key to be used for associating application-defined properties with streams within this connection.
+     *
+     * @param <K> The object type underlying the key.
+     * @param <V> The value type to be stored by this key.
+     */
+    interface Key<K, V> {
```

Comment:
Two requests:

1. Can we remove the `V` generic argument.  Seems like it is not needed and is not used in this interface.
2. Does it make sense to include more context to this name? `Key` -> `PropertyKey`?